variables.tf

```diff
@@ -43,72 +43,86 @@ variable "use_regional_s3_endpoint" {
 }
 
 variable "origin_bucket" {
+  type        = string
   default     = ""
   description = "Name of S3 bucket"
 }
 
 variable "origin_path" {
   # http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html#DownloadDistValuesOriginPath
+  type        = string
   description = "An optional element that causes CloudFront to request your content from a directory in your Amazon S3 bucket or your custom origin. It must begin with a /. Do not add a / at the end of the path."
   default     = ""
 }
 
 variable "origin_force_destroy" {
+  type        = bool
   default     = "false"
   description = "Delete all objects from the bucket  so that the bucket can be destroyed without error (e.g. `true` or `false`)"
 }
 
 variable "bucket_domain_format" {
+  type        = string
   default     = "%s.s3.amazonaws.com"
   description = "Format of bucket domain name"
 }
 
 variable "compress" {
+  type        = bool
   default     = "false"
   description = "Compress content for web requests that include Accept-Encoding: gzip in the request header"
 }
 
 variable "is_ipv6_enabled" {
+  type        = bool
```

Comment:
If these are type "bool" then the default needs to be unquoted.

Reply:
Unquoted type "bool"